adminlteui/templates/admin/change_form.html

```diff
@@ -129,5 +143,7 @@ <h4 class="box-title">
 <script src="{% static "admin/plugins/datatables/dataTables.bootstrap.min.js" %}"></script>
 <script>
 $('[required]').before('<span class="text-red" style="margin-left: -10px;">* </span>');
+$('.nav-tabs li:first').addClass('active');
+$('.tab-content div:first').addClass('active in');
```

Comment:
You could use `{% if forloop.first %}` to avoid using JS and set that in the `class` attribute directly.

Reply:
Good shout, will do